src/concern/mod.rs

```diff
@@ -165,10 +165,8 @@ pub struct WriteConcern {
 pub enum Acknowledgment {
     /// Requires acknowledgement that the write has reached the specified number of nodes.
     ///
-    /// Note: specifying 0 here indicates that the write is unacknowledged. Doing so means
-    /// the driver will not receive a response indicating whether an operation succeeded or failed.
-    /// It also means that the operation cannot be associated with a session. It is reccommended to
-    /// avoid using unacknowledged write concerns.
+    /// Note: specifying 0 here indicates that the write concern is unacknowledged, which is
```

Comment:
I meant to include this change in the last PR (I even commented that I did but forgot to push the commit), so I included it here.